src/mpid/ch4/include/mpidpre.h

```diff
@@ -183,7 +183,7 @@ typedef struct MPIDIG_req_ext_t {
         MPIDIG_get_req_t greq;
         MPIDIG_cswap_req_t creq;
         MPIDIG_acc_req_t areq;
-    };
+    } ext;
```

Comment:
This makes me wonder what does `ext` mean. I would much prefer simply name it `u`. We have 15 instances of using `u` for "anonymous-like" union usage, and we are used to it. I don't think we need introduce extra names unless we have to.

Reply:
Agreed. Let's use `u`.

Reply:
sure. What I worried about more is whether this is really necessary given that the impact is large.